source/app/viewoptions.cpp

```diff
@@ -17,6 +17,14 @@
   
 #include "viewoptions.h"
 
+static constexpr double MIN_ZOOM = 25;
+static constexpr double MAX_ZOOM = 300;
```

Comment:
I feel these should live in a more general place so that the playback widget could also appreciate them. Any suggestions?

Reply:
I think these could be fine as static constants on the `ViewOptions` class, and then the UI widgets can refer to those constants as needed.